package.json

```diff
@@ -15,7 +15,7 @@
     "lint": "node src/lint.js",
     "lint-fix": "node src/lint.js --fix",
     "test": "mocha",
-    "test-pr": "mocha --exclude test/fetch-info-w3c.js --exclude test/fetch-groups-w3c.js"
+    "test-pr": "mocha --exclude test/fetch-groups-gh.js"

```

Comment:
See comment on `fetch-groups-gh.js`. My understanding is that `test-pr` could be dropped as the job that runs on pull requests should now be able to run `test` directly.